modules/jt-comparison-of-jenkins-and-openshift-pipelines-concepts.adoc

```diff
@@ -38,16 +39,16 @@ You can initiate a PipelineRun or a TaskRun with a set of inputs such as paramet
 +
 [NOTE]
 ====
-In Jenkins, there is no direct equivalent of Tekton workspaces. You can think of the control node as a workspace, as it stores the cloned code repository, build history, and artifacts. In situations where a job is assigned to a different node, the cloned code and the generated artifacts are stored in that node, but the build history is maintained by the control node.
+In Jenkins, there is no direct equivalent of OpenShift Pipelines workspaces. You can think of the control node as a workspace, as it stores the cloned code repository, build history, and artifacts. In situations where a job is assigned to a different node, the cloned code and the generated artifacts are stored in that node, but the build history is maintained by the control node.
```

Comment:
Replace "control node" by "controller node" which is the more appropriate terminology.

Reply:
Thanks. Updated.